src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/Configuration/SelectAttributeConfigurationType.php

```diff
@@ -44,6 +45,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('max', NumberType::class, [
                 'label' => 'sylius.form.attribute_type_configuration.select.max'
             ])
+            ->addEventSubscriber(new ChangeStructureOfChoicesFormEventSubscriber())
```

Comment:
We should make a custom type for `choices` rather than make custom event subscriber changing the child of this form.